Formula/p/pistache.rb

```diff
@@ -0,0 +1,157 @@
+class Pistache < Formula
+  desc "Modern, fast, elegant HTTP + REST C++17 framework with pleasant API"
+  homepage "https://github.com/pistacheio/pistache"
+  url "https://github.com/pistacheio/pistache/archive/refs/tags/v0.4.26.tar.gz"
+  sha256 "29af6562547497acf6f49170661786fe8cf1ed3712ad80e69c53da4661c59544"
+  license "Apache-2.0"
+  head "https://github.com/pistacheio/pistache.git", branch: "master"
+
+  depends_on "cmake" => :build # Required so meson finds hinnant-date
+  depends_on "cpp-httplib" => :build
+  depends_on "googletest" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "brotli"
+  depends_on "howard-hinnant-date"
+  depends_on "libevent"
+  depends_on "openssl@3"
+  depends_on "rapidjson"
+  depends_on "zstd"
+
+  uses_from_macos "curl" => :build # For build on Linux; macOS ignores
```

Comment:
needs zlib for linux build (also removed the comment, as `uses_from_macos` already covered the semantics)

```suggestion
  uses_from_macos "curl" => :build
  uses_from_macos "zlib"
```

Reply:
OK, changed